### ISSUE TITLE
Handle pages with symbols in a reasonable way

### ISSUE THREAD

**Issue by ppannuto** [c4cs/c4cs.github.io#223]
Once upon a time they were named by what they do (e.g. ">" was redirect, though it should be something more like truncateAndRedirect), but that got tricky when using their names as handles/titles elsewhere.

Needs some thinking about.

Can uncomment the references in syllabus week3 when fixed.

**Comment by christyliu** [comment]
branching off of this, is the intended behavior for the >, |, and >> pages in the "scripting" section to redirect to /commands (which seems to only contain information for pipe)?

**Comment by ppannuto** [comment]
No.. this is an old problem, if you have a solution, awesome. The underlying issue is aesthetic. The goal would be to link files like https://github.com/c4cs/c4cs.github.io/blob/master/_commands/scripting/%3E.md as well, the challenge is that the syllabus picks up entries from the commands section (i.e. https://raw.githubusercontent.com/c4cs/c4cs.github.io/master/_syllabus/w17/week03.md ), and renders the list based on the string keys there (via https://github.com/c4cs/c4cs.github.io/blob/master/_layouts/syllabus.html#L72 ):

![image](https://cloud.githubusercontent.com/assets/339422/22660342/2109ef24-ec6f-11e6-8e9f-c30f4ebd685a.png)

So whatever escaping is done, ultimately the right thing needs to show up in the list of discussed commands on the syllabus. That's that hard part IIRC

I didn't have time to puzzle out the right way to do that, so you're not stomping on anyone's WIP if you've got a solution :)

**Comment by mterwill** [comment]
It looks like there's a `uri_escape` filter in Jekyll if that helps.
